config/routes.rb

```diff
@@ -21,9 +21,10 @@
     resources :lines, only: [:new, :create]
   end
   root :to => redirect('/users/sign_in')
-  devise_for :users, skip: [:registrations]
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
+  # devise_for :users, skip: [:registrations]
```

Comment:
There was some error with this stuff not commented out. I think it can get added back in just wanted to make sure the landscape was settled before I did.